docs/docs/administration/backup-and-restore.md

```diff
@@ -49,10 +52,11 @@ docker exec -t immich_postgres pg_dumpall --clean --if-exists --username=postgre
 ```
 
 ```powershell title='Restore'
-docker compose down -v  # CAUTION! Deletes all Immich data to start from scratch.
-# Remove-Item -Recurse -Force DB_DATA_LOCATION # CAUTION! Deletes all Immich data to start from scratch.
+docker compose down -v  # CAUTION! Deletes all Immich data to start from scratch
```

Comment:
Since we are moving from docker volume, this is not necessary true anymore, correct?

Reply:
Yes, I've just left it as before (not sure why it shows as an added line) - it's quite hard to explain the nuance of "if you installed before 1.102 it WILL delete your DB"...